src/pymatgen/analysis/elasticity/elastic.py

```diff
@@ -479,8 +479,7 @@ def from_pseudoinverse(cls, strains, stresses) -> Self:
             "questionable results from vasp data, use with caution."
         )
         stresses = np.array([Stress(stress).voigt for stress in stresses])
-        with warnings.catch_warnings():
```

Comment:
Looks like a bare `with warnings.catch_warnings():` (without any filter or such) is not doing anything